src/js/transfers/nep141~erc20/natural-erc20/getMetadata.js

```diff
@@ -19,7 +19,7 @@ async function getBalance (address, user) {
 
 const erc20Decimals = {}
 async function getDecimals (address) {
-  if (erc20Decimals[address]) return erc20Decimals[address]
+  if (erc20Decimals[address] !== undefined) return erc20Decimals[address]
```

Comment:
Is there a different edge case here ? The ` !== undefined` check is not done in natural-erc20/getName.js/getName() which is also called by natural-erc20/getMetadata.js/getErc20Data().
Seems like both should be the same

Reply:
`getName` always returns a non-blank string, so the `if (erc20Names[address])` check is sufficient. This one defaults to `0`, which is falsey, and so it needs the stricter `!== undefined`